### PR TITLE
Paul's contribution to the UG

### PR DESCRIPTION
**Pull request by paulktham**
Added description for:
- Minimum
- Add
- Parts of Quick Start
- Personal PPP

### CODE REVIEW

File: docs/team/paulktham.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### User Guide Contributions:
+- Along with the other group members we discuss the structure of our User Guide.
+    - Content Table.
+    - Explained how to run our application through Quick Start.
+    - Added `Add` and `Minimum` Command.
+
+### Developer Guide Contributions:
+- Along with the other group members we drew the different diagrams needed in our Developer guide.
+    - Drew the Architecture diagram as well as the sequence diagram found under StudentList.
```
Comment (Holy-An) [inline docs/team/paulktham.md:31]: Nice

File: docs/team/paulktham.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Summary of contributions
+[RepoSense (tP Code Dashboard)](https://nus-cs2113-ay2425s1.github.io/tp-dashboard/?search=paulktham&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2024-09-20)
+
+### v1.0 Enhancements: `Student` Class, `University` Class and `UniversityRepository` Class `PrintStudentList` Method Design Improvement
+- Student (`Student`) Development
+    - Created the class to represent one student applying for SEP, ensuring that it has the right information for the allocator to fairly decide which student is allocated to which University. I also had to think ahead with my team members to decide what helper functions they require so that we can implement other features seamlessly.
+
+- University (`University`) Development
+    - Created the University Class which holds the relevant information for each University available to NUS CEG students for exchange. Using this, I populated a class UniversityRepository which has a static HashMap which holds the information of all the Universities.
```
Comment (Holy-An) [inline docs/team/paulktham.md:17]: Interesting insights

File: docs/UserGuide.md [hunk 3]
```diff
@@ -97,6 +125,7 @@ Format: `revert`
 | Statistic | ``stats -avggpa UNI_INDEX``  or ``stats -mingpa UNI_INDEX`` <br> e.g. ``stats -mingpa 42`` |
 | viewQuota | ``viewQuota UNI_INDEX`` <br> e.g. ``viewQuota 42``           |
 | Allocate  | `allocate`                                                   |
+| Minimum   | `minimum 4.0`                                                |
```
Comment (Holy-An) [inline docs/UserGuide.md:128]: Oof

**Review by Holy-An — APPROVED** [review]

**Review by ehz0ah — APPROVED** [review]
Very well made 😺

**Pull request merged**